Shared/ThresholdKeyView.swift

```diff
@@ -16,9 +16,13 @@ struct ThresholdKeyView: View {
     @State private var resetAccount = true
     @State var threshold_key: ThresholdKey!
     @State var shareCount = 0
-
+    @State private var isLoadingAddPassword = false
+    @State private var isLoadingChangePassword = false
+    @State private var showInputPasswordAlert = false
+    @State private var password = ""
```

Comment:
We probably do not want to keep a password in the app state at any point, rather have the user input it as needed.

Reply:
@metalurgical  This state is binded with input field  `SecureField("New Password", text: $password)`. I am new to the lang, Is there any better way to handle this, without having to keep password in state ?  Please suggest

Reply:
Hmm, I see. In that case just reset the password var back to it's default value after using it, that should be enough.